dockerfiles/run/guice/cassandra-ldap/destination/conf/listeners.xml

```diff
@@ -21,6 +21,10 @@
 <listeners>
   <listener>
     <class>org.apache.james.mailbox.spamassassin.SpamAssassinListener</class>
```

Comment:
while we are at it, I would rather not put it in default listeners.xml because i'm not sure people will deploy a SpamAssassin very often

Reply:
It's harmless as if the connection can not be established, the listenenr is skept.

Reply:
I don't agree, logs are always a source of support questions, the less you have, the better